sphinx/application.py

```diff
@@ -650,12 +987,34 @@ def add_stylesheet(self, filename, alternate=False, title=None):
 
     def add_latex_package(self, packagename, options=None):
         # type: (unicode, unicode) -> None
+        """Register a package to include in the LaTeX source code.
+
+        Add *packagename* to the list of packages that LaTeX source code will
+        include.  If you provide *options*, it will be taken to `\\usepackage`
+        declaration.
+
+        .. code-block:: python
+
+           # => \\usepackage{mypackage}
+           app.add_latex_package('mypackage')
+           # => \\usepackage[foo,bar]{mypackage}
+           app.add_latex_package('mypackage', 'foo,bar')
```

Comment:
IMO, it is better to place result line (`# =>`) after this line.

Reply:
Done